docs/shared-content/installation/rad-cli/install-rad-cli.md

```diff
@@ -26,9 +28,11 @@ wget -q "https://get.radapp.dev/tools/rad/install.sh" -O - | /bin/bash
 {{< edge >}}
 To install the latest edge version:
 
-```bash
-wget -q "https://radiuspublic.blob.core.windows.net/tools/rad/install.sh" -O - | /bin/bash -s edge
-```
+1. Visit the [GitHub Actions runs](https://github.com/radius-project/radius/actions/workflows/build.yaml?query=branch%3Amain+event%3Apush)
```

Comment:
since the instructions/content are the same for Windows, macOS and Linux, do we need separate tabs for them?

Reply:
Yeah we'll keep them for now. We're going to switch back to os-specific instructions with the Oras client once we start pushing that to GHCR